src/layer/AbstractDEMLayer.ts

```diff
@@ -116,3 +146,5 @@ export class AbstractDEMLayer extends AbstractSentinelHubV3Layer {
     return tiles.map(tile => moment.utc(tile.sensingTime).toDate());
   }
 }
+
+const isBooleanNull = (value: boolean): boolean => value === null || value == undefined;
```

Comment:
I think `value` is not  a `boolean`, it's of types `null` or `undefined`.  I would assume `isDefined` or smth would be a more appropriate name.

Reply:
Function was renamed to `isDefined`